test/__snapshots__/index.test.js.snap

```diff
@@ -2728,9 +2728,10 @@ ts-custom-declaration
     index.js.map
     index.umd.js
     index.umd.js.map
+  node_modules
```

Comment:
```suggestion
```

Reply:
Clicked "Commit suggestion" too fast there. What was not working w/ the CI? Seemed to be passing w/out any issues?

rpt2 creates some cache files in `node_modules`, so I believe it's needed.

Matches the other declaration test:

https://github.com/developit/microbundle/blob/b1a637486234a2ae784ccf0c512321e2d3efef7c/test/__snapshots__/index.test.js.snap#L2775-L2795